api/model.go

```diff
@@ -542,6 +543,16 @@ func toSettingCollection(settings []*longhorn.Setting) *client.GenericCollection
 	return &client.GenericCollection{Data: data, Collection: client.Collection{ResourceType: "setting"}}
 }
 
+func getReplicaName(address string, vrs []*longhorn.Replica) string {
+	for _, r := range vrs {
+		if strings.Contains(address, r.Status.IP) {
```

Comment:
Need a more precise matching, so `tcp://1.2.3.45`, wont match `1.2.3.4` instead of `1.2.3.45`.